src/js/components/Box/Box.js

```diff
@@ -27,7 +27,7 @@ class Box extends Component {
   };
 
   getChildContext() {
-    const { grommet } = this.context;
+    const { grommet = {} } = this.context;
```

Comment:
Given `grommet` is required, do we need this initializer?

Reply:
nope :)